hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -327,3 +327,48 @@ def test_manual_remove_element(self):
         assert (
             sorted(s._xray_markers.keys()) ==
             ['Zn_Ka', 'Zn_La'])
+
+
+@lazifyTestClass
```

Comment:
Thanks for the explanations @to266!
1. I was missing the point of the `setup_method`, so yes, the exact same tests are run twice... But after defining the signals in `setup_method`, the tests run correctly, except for the one where some data are changed manually, which is normal.
2. I am not sure to understand here, but it looks to me that rebinning lazy signals works and after fixing the tests they also work. The following works:
```python
import numpy as np
import hyperspy.api as hs

s = hs.signals.Signal1D(np.arange(100).reshape((10, 10)))
sl = s.as_lazy()

# linear interpolation
sl2 = sl.rebin((3,4))
# fast integer rebin
sl3 = sl.rebin((2,2))
```